lib/MusicBrainz/Server/Report/PossibleCollaborations.pm

```diff
@@ -20,8 +20,7 @@ sub query {
                 WHERE 
                     l_artist_artist.entity1=artist.id
                     AND link_type.name IN
-                      ('collaboration', 'member of band')
-            )
+                      ('collaboration', 'conductor position', 'founder', 'member of band', 'subgroup')            )
```

Comment:
Report’s description should be updated accordingly.
By the way, I cannot understand why it mentions URL relationships.

Reply:
Fair enough (but it doesn't mention URL relationships?)

Reply:
Oops, I was looking at CollaborationRelationships instead of PossibleCollaborations!
Please link to the related MB report in further PRs about MB reports.